zoho_mcp/resources.py

```
@@ -46,509 +42,213 @@ async def get_dashboard_summary() -> Resource:
             raise
     
-    @mcp.resource("invoice://overdue")
-    async def get_overdue_invoices() -> Resource:
+    @mcp.resource("invoice://overdue", name="Overdue Invoices", title="Overdue Invoices List", description="List of all invoices that are past their due date", mime_type="text/plain")
+    async def get_overdue_invoices() -> str:
         """Get list of overdue invoices."""
         logger.info("Fetching overdue invoices")
         
         try:
             # Fetch overdue invoices
             params = {
```

Ruff (0.12.2)

Local variable `params` is assigned to but never used

Remove assignment to unused variable `params`

(F841)